servers/republik/modules/crowdfundings/graphql/resolvers/Membership.js

```diff
@@ -178,6 +178,16 @@ const membershipResolver = {
     const type = await membershipResolver.type(membership, null, context)
 
     return type.interval === 'year'
+  },
+
+  async canManuallyAddPeriod ({ subscriptionId: stripeSubscriptionId, active }, x, { pgdb }) {
```

Comment:
There is probably little need to alias `subscriptionId` to `stripeSubscriptonId`.

Reply:
@patrickvenetz with regard to the `stripeSubscriptionId`: The idea was to put an emphasis on the fact, that this is not the `id` of the `subscriptions` table, which I initially thought and lost time with. Obviously, this renaming should happen on the DB field, not here.

Given this argument, would you still not using the alias?

Reply:
Haven't even thought of it, and find that a rather compelling argument. Especially since you fell for it.

(I'd probably still not use the alias, arguing somewhat on the scope in which this resolver is to be found and what it handles, both membershipy stuff. Also: worked-too-long-with-that-subscription-ID-bias.)

Not taking a stand here, `stripeSubscriptionId` is good to and it reflects very much what it is.